requirements.txt

```diff
@@ -13,3 +13,5 @@ numpy==1.15.1
 databases==0.2.2
 aiosqlite==0.10.0
 discord.py[voice]==1.3.3
+feedparser >= 6.0.2
+asyncio >= 3.4.3
```

Comment:
So I've used this library elsewhere and I was wondering why I didn't already have it in requirements.txt. Based on [this](https://pypi.org/project/asyncio/), its only needed for python 3.3, anything higher and it will be included in the std library by default. I'm worried about it installing an older version than whats currently in the std library so I'd recommend just removing it from the requirements.txt here.